lint/linter.go

```diff
@@ -106,7 +106,7 @@ func (l *Linter) Lint(packages [][]string, ruleSet []Rule, config Config) (<-cha
 		wg.Add(1)
 		go func(pkg []string, gover *goversion.Version) {
 			if err := l.lintPackage(pkg, gover, ruleSet, config, failures); err != nil {
-				fmt.Fprintln(os.Stderr, err)
+				fmt.Fprintln(os.Stderr, "Error during linting:"+err.Error())
 				os.Exit(1)
 			}
 			wg.Done()
```

Comment:
Thanks for the changes you just made on errgroup

I just found another one here that could be refactored with errgroup.

I noticed it because you somehow tried to wrathe error message at an higher level

Reply:
I reported it because fixing a deep os.Exit, is almost as the same scope of your PR. Panic and os.Exit should be removed.

I can of course open a PR if you prefer

Reply:
I reported it because fixing a deep os.Exit, is almost as the same scope of your PR. Panic and os.Exit should be removed.

I can of course open a PR if you prefer

Reply:
ok ill try to refactor func (l *Linter) Lint() with errgroup way :)